{"name": "Load more", "status": "broken", "statusDetails": {"message": "TypeError: WebDriver.__init__() got an unexpected keyword argument 'executable_path'", "trace": "request = <SubRequest 'driver' for <Function test_upload_1error_file>>\n\n    @pytest.fixture(scope=\"function\", autouse=True)\n    def driver(request):\n        options = Options()\n        perfs = {\"download.defoult_directory\": f\"{os.getcwd()}/downloads\"}\n        #options.add_argument(\"--headless\")\n        options.add_experimental_option(\"excludeSwitches\", ['enable-automation'])\n        options.add_experimental_option(\"useAutomationExtension\", False)\n        options.add_experimental_option(\"prefs\", perfs)\n        options.add_argument(\"--window-size=1920,1080\")\n        options.add_argument(\"--no-sandbox\")\n        options.add_argument(\"--disable-dev-shm-usage\")\n        options.add_argument(\"--ignore-certificate-errors\")\n        ser = Service(f\"{os.getcwd()}/base/chromedriver\")\n>       driver = webdriver.Chrome(service=Service(ChromeDriverManager().install()), executable_path=ser, options=options)\nE       TypeError: WebDriver.__init__() got an unexpected keyword argument 'executable_path'\n\nconftest.py:25: TypeError"}, "start": 1718677092219, "stop": 1718677092219, "uuid": "6c4cc97d-0cf9-403d-b0e2-6f175f7c09b5", "historyId": "b876855393093e4f206ee3077aea6915", "testCaseId": "b876855393093e4f206ee3077aea6915", "fullName": "tests.test_bev.TestBEV#test_upload_1error_file", "labels": [{"name": "feature", "value": "BEV Functionality"}, {"name": "severity", "value": "Critical"}, {"name": "parentSuite", "value": "tests"}, {"name": "suite", "value": "test_bev"}, {"name": "subSuite", "value": "TestBEV"}, {"name": "host", "value": "34d0473f3ec0"}, {"name": "thread", "value": "1-MainThread"}, {"name": "framework", "value": "pytest"}, {"name": "language", "value": "cpython3"}, {"name": "package", "value": "tests.test_bev"}]}